src/linttest/regression_test.go

```diff
@@ -1243,3 +1243,31 @@ function f($x) {
 }
 `)
 }
+
+func TestIssue406(t *testing.T) {
+	test := linttest.NewSuite(t)
+	test.AddFile(`<?php
+
+interface A {
```

Comment:
Another test case idea: method and constant without explicit access modifier.
PHP implies `public` for these.
These cases should not give a warning (we may give a warning about missing access modifier later in a separate check, so it can be disabled separately).